skietbaan_fe/src/components/Login.js

```diff
@@ -84,7 +78,6 @@ class App extends Component {
     if (this.state.invalidUsername) {
       invalidUsernameMessage = <div className="invalid-message">Please enter your username</div>;
```

Comment:
While this is already merged code. Please do read up on some of the sign in best practices. One of them being that you should never give the user info on what was wrong with their credentials. It can be used to determine if users exist and then to brute force passwords for those users. Rather change those states 3 to a single invalid credentials one and give an appropriate message. Server should then have logs on what actually happened.

Reply:
This validation just ensures the field isnt empty

Reply:
Ah miss read. Its fine then. But still worth reading up when you handle the login request and it returns with an error. Keeping in mind that if server sends you anything aside form invalid creds for (user not found, invalid password, invalid username, etc). You should have them change it

Reply:
Ok thanks, will do so